src/device_tracker.cc

```diff
@@ -105,6 +105,14 @@ void DeviceTracker::Initialize(const cbor::Value::ArrayValue& versions,
       }
     }
   }
+
+  std::vector<std::string> default_true_options = {"up"};
+  for (const std::string& option : default_true_options) {
+    auto iter = options.find(cbor::Value(option));
+    if (iter == options.end()) {
```

Comment:
There seems to be a `contains` method, so `!options.contains(cbor::Value(option))` should be an equivalent condition.

Reply:
`std::map` has `contains` since C++20, but we use C++17. So this should be the best we can do?

Reply:
Ah ok it's `std::map`. I don't know where to get this information. I just tried to Google `cbor::Value::MapValue` and found [this definition](https://chromium.googlesource.com/chromium/src/+/master/components/cbor/values.h#96) which seems to have [a `contains` method](https://chromium.googlesource.com/chromium/src/+/master/base/containers/flat_map.h#157). Sounds good if it doesn't compile with `contains`.

Reply:
When I ported the CBOR parser out of Chromium, I replaced some data structures with standard replacements. You could check `third_party/`. The compiler agrees there is no `contains`.

(btw: The port was done when we still used C++11, so I think there are some possible improvements there. But unrelated and it shouldn't be important right now.)

Reply:
Oh I see, I missed that `third_party` was not "generated" :D Makes sense, thanks for the info, now it's simpler to know where to check.